### ISSUE TITLE
shield/hull regeneration in port 

### ISSUE THREAD

**Issue by wambat1** [endless-sky/endless-sky#1201]
ships parked with shield generators (provided they are not disabled) ought to disembark with fully recharged shields. likewise with hull regenerators. landing and taking off takes a whole day in game time; any self repairing equipment should have completed repairs in that time.

**Comment by Alkallid** [comment]
And with ramscoops, as tge sun is probably going to shine on the planet for a while, and you could use that to refuel. It would mess up other stuff, however, because there's every ship's mini version.

**Comment by endless-sky** [comment]
Ramscoops work by "harvesting the solar wind," not sunlight, so they presumably wouldn't work planetside.

**Comment by Alkallid** [comment]
Pseudo-science it then m8y. I need to leave Host soon...

**Comment by jafdy** [comment]
The solar wind would be blocked by the planets magnetic field. So tectonicly active planets won't have a solar wind blowing on them.
On the other hand if it harvested sunlight you could stick a light bulb in front of it to refill your tank.

**Comment by Bladewood** [comment]
> stick a light bulb in front of it to refill your tank
If humanity is powerful enough to convert solar energy directly to mass in the form of fuel...

**Comment by wambat1** [comment]
"solar wind" is (massive) particles escaping a star, some percentage of which will be usable as fuel, or can be converted to fuel. it is not the sunlight; i.e., non-massive electromagnetic radiation. 

    On Monday, June 6, 2016 3:55 AM, Bladewood <notifications@github.com> wrote:
 

 
stick a light bulb in front of it to refill your tank
If humanity is powerful enough to convert solar energy directly to mass in the form of fuel...

**Comment by endless-sky** [comment]
Fixed in e9ee546.
